kaylaewood/RefactorTractor
# Change user functionality

#### What's this PR do?
Now, the user can change users by using the form. The form checks that the information input by the user is correct before updating the DOM. I also added a "Cancel" button on all forms to increase UX.
#### Where should the reviewer start?
Start by using the 'Change User' form on the browser and then look at the scripts file to see the code.
#### How should this be manually tested?
Ensure that the form does change the user without errors in the console. You can check that it's returning the correct user with these three: 
1 - Louisa
2 - Jarvis
5 - Mae
Also, try to input invalid ID numbers (any thing outside of 1-50, any letters, etc.).
